cinema/views.py

```diff
@@ -1,10 +1,13 @@
 from datetime import datetime
 
 from django.db.models import F, Count
-from rest_framework import viewsets
+from rest_framework import viewsets, mixins
+from rest_framework.authentication import TokenAuthentication
 from rest_framework.pagination import PageNumberPagination
+from rest_framework.permissions import (IsAuthenticated)
```

Comment:
The import statement for `IsAuthenticated` should not have parentheses. Correct it to: `from rest_framework.permissions import IsAuthenticated`.